src/runtime/exceptions.cs

```diff
@@ -82,6 +82,13 @@ public new static IntPtr tp_str(IntPtr ob)
             }
             return Runtime.PyUnicode_FromString(message);
         }
+
+        public static int tp_init(IntPtr ob, IntPtr args, IntPtr kwds)
+        {
+            Exceptions.SetArgsAndCause(ob);
```

Comment:
What comment?

Reply:
NIT: You seem to have moved this from `CLRObject`, and removed
```
// Fix the BaseException args (and __cause__ in case of Python 3)
// slot if wrapping a CLR exception
```

Reply:
The comment on `SetArgsAndCause` won't be enough?